resolwe/toolkit/tests/test_dirs.py

```diff
@@ -0,0 +1,61 @@
+# pylint: disable=missing-docstring
+import os
+import unittest
+
+from resolwe.flow.models import Data
+from resolwe.test import ProcessTestCase, tag_process, with_docker_executor
+
+
+class DirsProcessTestCase(ProcessTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'files'))
+
+        test_case = unittest.TestCase()
+        self.assertRaisesRegex = test_case.assertRaisesRegex  # pylint: disable=invalid-name
+
+    @with_docker_executor
+    @tag_process('upload-dir')
+    def test_upload_dir(self):
```

Comment:
This test tests too many things. You just need to make sure that process works as expected, so chose only one assertion function. All other cases should be tested in `test_assert_dir_structure`, where assertion functions are tested.